rama-http/Cargo.toml

```diff
@@ -30,6 +30,7 @@ async-compression = { workspace = true, features = [
 base64 = { workspace = true }
 bitflags = { workspace = true }
 bytes = { workspace = true }
+chrono = "0.4.39"
```

Comment:
You need to add this to `./Cargo.toml` (the root file) and than use here `chrome = { workspace = true }`